docs/manuals/source/TasksAndConcepts/BareosConsole.rst

```diff
@@ -1331,48 +1332,86 @@ status
    days=number
       of days shows only the number of days in the scheduler preview. Positive numbers show the future, negative numbers show the past. days can be combined with the other selection criteria. days= can be combined with the other selection criteria.
 
-   In case you are running a maintained version of Bareos, the command :bcommand:`status subscriptions` (:sinceVersion:`12.4.4: status subscriptions`) can help you to keep the overview over the subscriptions that are used.
-
-   To enable this functionality, just add the configuration :config:option:`dir/director/Subscriptions`\  directive and specify the number of subscribed clients, for example:
-
-   .. code-block:: bareosconfig
-      :caption: enable subscription check
-
-      Director {
-         ...
-         Subscriptions = 50
-      }
+status subscriptions
+   In case you have a service contract for Bareos, the command :bcommand:`status subscriptions` (:sinceVersion:`12.4.4: status subscriptions`) can help you to keep the overview over the subscriptions that are used.
```

Comment:
```suggestion
   In case you have a service contract for Bareos, the command :bcommand:`status subscriptions`  can help you to keep the overview over the subscriptions that are used.
```